terraform/catalogue_api/cluster.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_ecs_cluster" "catalogue_api" {
+  name = "catalogue-api-20210426"
```

Comment:
Is it too late to add dashes to the date? I find `2021-04-26` much easier to read than `20210426`.

Reply:
Nope not too late, will do